### ISSUE TITLE
Fix "load more" button to look for (and retrieve) active posts only

### ISSUE THREAD

**Issue by jdunk** [jmocko/cuecount#2]
## (From jmocko's email to me) :

The "load more" button at the bottom of the Feed page looks for the 5 earlier posts based on their id, in order, and then displays them.

However, because posts are "ended" and therefore hidden from the feed, the "load more" button might only actually return 3 posts (if the ids of two of the posts were associated with ended polls.)

Sometimes it'll return no posts and therefore conclude that there are no more in the database, so it'll show a message and cut off users from seeing anymore.

so if 5 or more ids in a row are associated with ended posts the feed will be cut off.

Basically, we need a new way of calling the next five posts that's not based on sequential id numbers, but on the next 5 'active' posts.
